sdk/healthinsights/azure-healthinsights-radiologyinsights/azure/healthinsights/radiologyinsights/_operations/_patch.py

```diff
@@ -7,8 +7,168 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+from io import IOBase
+import json
+import sys
+from typing import Any, Callable, Dict, IO, List, Optional, Type, TypeVar, Union, cast, overload
+
+from azure.core.exceptions import (
+    ClientAuthenticationError,
+    HttpResponseError,
+    ResourceExistsError,
+    ResourceNotFoundError,
+    ResourceNotModifiedError,
+    map_error,
+)
+from azure.core.pipeline import PipelineResponse
+from azure.core.polling import LROPoller, NoPolling, PollingMethod
+from azure.core.polling.base_polling import LROBasePolling
+from azure.core.rest import HttpRequest, HttpResponse
+from azure.core.tracing.decorator import distributed_trace
+from azure.core.utils import case_insensitive_dict
+
+from .. import models as _models
+from .._model_base import SdkJSONEncoder, _deserialize
+from .._serialization import Serializer
+from .._vendor import RadiologyInsightsClientMixinABC
+from ._operations import RadiologyInsightsClientOperationsMixin as GeneratedRadiologyInsightsClientOperationsMixin
+
+if sys.version_info >= (3, 9):
+    from collections.abc import MutableMapping
+else:
+    from typing import MutableMapping  # type: ignore  # pylint: disable=ungrouped-imports
+JSON = MutableMapping[str, Any]  # pylint: disable=unsubscriptable-object
+T = TypeVar("T")
+ClsType = Optional[Callable[[PipelineResponse[HttpRequest, HttpResponse], T, Dict[str, Any]], Any]]
+
+
+
+
+
+
+class RadiologyInsightsClientOperationsMixin(GeneratedRadiologyInsightsClientOperationsMixin):
+
+    @overload
+    def begin_infer_radiology_insights(
+        self,
+        id: str,
+        resource: _models.RadiologyInsightsJob,
+        *,
+        expand: Optional[List[str]] = None,
+        content_type: str = "application/json",
+        **kwargs: Any
+    ) -> LROPoller[_models.RadiologyInsightsInferenceResult]:...
+        
+
+    @overload
+    def begin_infer_radiology_insights(
+        self,
+        id: str,
+        resource: JSON,
+        *,
+        expand: Optional[List[str]] = None,
+        content_type: str = "application/json",
+        **kwargs: Any
+    ) -> LROPoller[_models.RadiologyInsightsInferenceResult]:...
+        
+
+    @overload
+    def begin_infer_radiology_insights(
+        self,
+        id: str,
+        resource: IO[bytes],
+        *,
+        expand: Optional[List[str]] = None,
+        content_type: str = "application/json",
+        **kwargs: Any
+    ) -> LROPoller[_models.RadiologyInsightsInferenceResult]:...
+        
+
+
+    @distributed_trace
+    def begin_infer_radiology_insights(
```

Comment:
I think you also need to update your method documentation to indicate the updated return type. Plus since your docs were auto-generated and are a few thousand lines long, it would be good to remove example input block.

Reply:
FYI I opened an issue for the tsp python emitter to fix this so that you dont get super long docs in future releases. You'll still need to patch the docs since they mention RadiologyInsightsJob instead of RadiologyInsightsInferenceResult